index.js

```diff
@@ -1,22 +1,74 @@
+// const token = '824082b02e4cf440e7352c08d3ae269fd8f1f83c';
```

Comment:
You should revoke this token, someone could use it to take over your account 👼

Reply:
Hey! Thanks for this. :) I revoked it after submitting. Github auto-updates the secret too if you push it remotely, which is pretty cool!